invenio_cli/helpers/package_managers.py

```diff
@@ -171,3 +178,186 @@ def start_activated_subshell(self) -> List[str]:
         # Also, it has a good chance of not properly setting a PS1...
         shell = os.getenv("SHELL")
         return [shell, "-c", f"source .venv/bin/activate; exec {shell} -i"]
+
+
+@dataclass
+class AssetsFunction:
+    """Function to run an assets command with `assets_pkg` and `module_pkg`."""
+
+    function: Callable[[NPMPackage, NPMPackage], ProcessResponse]
```

Comment:
the dataclass wants to have _some_ type annotation, but `Callable` should do the trick already.
i have no strong attachment to this type hint.